### PR TITLE
feat: reset fields after transfer is complete 

### LINKED ISSUE #133: Source network gets reset on starting new transfer
## Expected Behavior
If wallet is still connected we shouldn't reset selected source network.

## Current Behavior
We reset selected source network which ends us in unsupported state where wallet is connected before source network is selected

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Version

Sygma commit (or docker tag):
sygma-solidity version:
sygma-substrate version:
Go version:

### PR DESCRIPTION
**Pull request by Lykhoyda**
closes #133

### CODE REVIEW

File: packages/widget/src/components/common/dropdown/dropdown.ts [hunk 1]
```diff
@@ -36,12 +36,32 @@ export class Dropdown extends BaseComponent {
...
+
   @state()
   selectedOption: DropdownOption | null = null;
 
   @property({ attribute: false })
   onOptionSelected: (option?: DropdownOption) => void = () => {};
 
+  constructor() {
+    super();
+    this._setPreselectedOption();
```
Comment (mpetrunic) [inline packages/widget/src/components/common/dropdown/dropdown.ts:50]: isn't this suppose to be done on connected callback?

**Review by mpetrunic — APPROVED** [review]

File: packages/widget/src/components/common/dropdown/dropdown.ts (hunk 1 above)
Comment (Lykhoyda) [reply packages/widget/src/components/common/dropdown/dropdown.ts]: yeah, I can move it to the connected callback as well 👍

**Review by mpetrunic — APPROVED** [review]

File: packages/widget/src/controllers/transfers/fungible-token-transfer.ts [hunk 2]
```diff
@@ -105,8 +105,10 @@ export class FungibleTokenTransferController implements ReactiveController {
     this.host.requestUpdate();
   }
 
-  reset(): void {
-    this.sourceNetwork = undefined;
+  reset({ omitSourceNetworkReset } = { omitSourceNetworkReset: false }): void {
```
Comment (wainola) [inline packages/widget/src/controllers/transfers/fungible-token-transfer.ts:108]: if this is false by default, wouldn't make sense to define it as optional parameter and type cast the undefined? 🤔

**Review by wainola — CHANGES_REQUESTED** [review]

**Review by wainola — COMMENTED** [review]
Left one small comment

**Review by wainola — APPROVED** [review]

**Pull request merged**